src/configs/templates/apl-analog-clock.json

```diff
@@ -0,0 +1,165 @@
+{
+    "type": "APL",
+    "version": "1.6",
+    "__comment__": "Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.\nSPDX-License-Identifier: LicenseRef-.amazon.com.-AmznSL-1.0\nLicensed under the Amazon Software License  http://aws.amazon.com/asl/",
```

Comment:
Is there any other way to add this license? since adding it here will cause suggester validation warning.

Reply:
It is a json file, so cannot add regular comments like js file. I will create a followup task to filter out warnings on "__comment__".

Reply:
maybe cjson would be an option? https://www.npmjs.com/package/cjson

Reply:
Thanks for the suggestion, will consider it.